xrootd/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright ©2018 The go-hep Authors. All rights reserved.
```

Comment:
this file isn't needed. there's already a `LICENSE` file at the root of the repo.